CONTRIBUTING.md

```diff
@@ -36,113 +37,86 @@ Some of these guidelines are not followed in the oldest code in the repository,
 1. One statement per line, to make semi-automatic processing and reading of diffs much easier.
 2. Boolean Variables should use `stdbool.h`'s names `true` and `false`, not an integer ([reference](https://github.com/fontforge/fontforge/issues/724)).
 3. `return` statements should be inline with the indentation level they are being put on; don't put them at the left margin as in much of the existing code ([reference](https://github.com/fontforge/fontforge/issues/1208)).
-4. Use POSIX/gnulib APIs in preference to glib, e.g. `strdup` instead of `g_strdup` and `xvasprintf` instead of `g_printf_strdup`. 
-This minimizes the impact of non-standard types and functions.
+4. Use POSIX/gnulib APIs in preference to glib, e.g. `strdup` instead of `g_strdup` and `xvasprintf` instead of `g_printf_strdup`. This minimizes the impact of non-standard types and functions.
```

Comment:
Given I want to remove gnulib, I'd actually prefer if we didn't use gnulib specific stuff like `xasprintf`

Reply:
Yes. Let's remove all references to gnulib.